client/src/app/users/name.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { FormControl } from '@angular/forms';
+
+export class NameValidator {
```

Comment:
Nice to have included an example of a custom validator. Your comments, though, imply that it's not "working as it should"? Does it not actually work? Are there issues we should try to address?